src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,41 +1,82 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { useAppSelector } from '../../app/store';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
```

Comment:
Verify that the `setCurrentTodo` action is correctly defined in the `../../features/currentTodo` module. It should be an action creator that updates the current todo in the Redux store.